packages/ingest/parse-pdr.js

```diff
@@ -40,6 +40,40 @@ function getItem(spec, pdrName, name, must = true) {
   return null;
 }
 
+/**
+ * Validate parsedSpec matches expectations
+ *
+ * @param {Object} spec - parsed spec
+ * @returns {Object} validated spec
+ * @throws {Error}
+ */
+function validateParsedSpec(spec) {
+  const parsedSpec = Object.assign({}, spec);
```

Comment:
Why are we passing in the whole spec if we're just validating the checksum properties?